proxy/proxy/spacewalk-proxy.spec

```diff
@@ -43,7 +43,7 @@ Requires:       httpd
 %if 0%{?pylint_check}
 BuildRequires:  spacewalk-python2-pylint
 %endif
-BuildRequires:  mgr-push >= 4.0.0
+BuildRequires:  mgr-push >= 4.1.0
```

Comment:
Why?

Reply:
Well, why we had `4.0.0` in the first place?

My assumption is that we always want the proxy to use mgr-push from the same SUSE manager version. If that's wrong I can rollback this change.

Reply:
No, we should only add versioned dependencies when they have a real reason.
Please revert to 4.0.0